ui/admin/app/controllers/scopes/scope/aliases/index.js

```diff
@@ -8,7 +8,6 @@ import { service } from '@ember/service';
 import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 import { loading } from 'ember-loading';
-import { debounce } from 'core/decorators/debounce';
```

Comment:
None blocking comment:
Out of the PR context but I notice that after this fix, the only place where we still using the debounce decorator is on the dropdown component within core, [here code](https://github.com/hashicorp/boundary-ui/blob/main/addons/core/addon/components/dropdown/index.js#L42), should we create a Jira task to try refactor that one with ember-concurrency as well? 
I think if we do that, we can get rid of the debounce decorator. What are your thoughts?

Reply:
I had the same question but adding a task here didn't look like it would really get us much since that one was just dealing with filtering out the dropdown all in memory (and tests seemed fine). I could just change it to a task tho if we feel like we should get rid of the decorator and we make the decision to always use them from now on in these scenarios. 

I'm fine with either way though, not sure what others think.